core/templates/pages/admin-page/misc-tab/admin-misc-tab.component.html

```diff
@@ -285,9 +285,9 @@ <h3>Lookup exploration interactions</h3>
   </div>
 
   <div>
-    <p *ngIf="explorationInteractions.length > 0">Exploration IDs:</p>
```

Comment:
explorationInteractionIds (be careful about naming, here and elswhere)

Reply:
Done.